### ISSUE TITLE
Add frame sequential output

### ISSUE THREAD

**Issue by kinng** [gkv311/sview#123]
I see there is frame sequential input.  Cam frame sequential output be added?

**Comment by gkv311** [comment]
I don't quite understand the question. Are you looking for a Shutter Glasses output? Have you tried it?

**Comment by kinng** [comment]
Yes I have tried it and it worked, albeit not very stable.  Is Shutter Glasses output the same as frame sequential?

**Comment by gkv311** [comment]
I guess it is. This output requires driver support - OpenGL Quad Buffer or NVIDIA extension. Without it, software emulated page flipping couldn't garantee stable output.

**Comment by kinng** [comment]
Yes I only could use the software emulated buffer.  Is OpenGL Quad Buffer an additional download?

**Comment by gkv311** [comment]
OpenGL QuadBuffer is a feature of your graphic driver. You should check vendor's documentation / contact their support to see if your hardware and software configuration has such feature (system + driver + videocard + display/glasses).
